### PR TITLE
new freature-1 added

### PR DESCRIPTION
**Pull request by Bijay215661**
some code

### CODE REVIEW

File: calculater.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
 again =True
+print("1234")
+print("456")
+print("34")
```
Comment (sagarPakhrin) [inline calculater.py]: please remove unnecessary print

File: calculater.py [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 # Mathmetical types
-ad =  "1"
-sb = "2"
-mul = "3"
-dev = "4"
+ad =  '1'
+sb = '2'
+mul = '3'
+dev = '4'
```
Comment (sagarPakhrin) [inline calculater.py]: try using a dictionary instead of multiple variables

File: calculater.py [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 
 #print function
 print(ad, "Addition")
 print(sb, "Subraction")
 print(mul,"Multiplication")
 print(dev, "Division")
-choise = input("Enter your chpoise : ")
+choise = input("Enter your choise : ")
```
Comment (sagarPakhrin) [inline calculater.py:17]: good catch

**Review by sagarPakhrin — CHANGES_REQUESTED** [review]

File: calculater.py (hunk 1 above)
Comment (Bijay215661) [reply calculater.py]: I removed

File: calculater.py (hunk 3 above)
Comment (Bijay215661) [reply calculater.py:17]: thanks

**Review by sagarPakhrin — APPROVED** [review]
lgtm

**Pull request merged**